e2e/aws/databases_test.go

```diff
@@ -315,3 +323,70 @@ func waitForDatabases(t *testing.T, auth *service.TeleportProcess, wantNames ...
 		}
 	}, 1*time.Minute, time.Second, "waiting for the database service to heartbeat the databases")
 }
+
+// dbAdminInfo contains common info needed to connect as an admin user via
+// password auth.
+type dbAdminInfo struct {
+	address,
+	username,
+	password string
+	port int
+}
+
+func connectAsPostgresAdmin(t *testing.T, ctx context.Context, info dbAdminInfo, dbName string) *pgx.Conn {
+	pgCfg, err := pgx.ParseConfig(fmt.Sprintf("postgres://%s:%d/?sslmode=verify-full", info.address, info.port))
+	require.NoError(t, err)
+	pgCfg.User = info.username
+	pgCfg.Password = info.password
+	pgCfg.Database = dbName
+	pgCfg.TLSConfig = &tls.Config{
+		ServerName: info.address,
+		RootCAs:    awsCertPool.Clone(),
+	}
+
+	conn, err := pgx.ConnectConfig(ctx, pgCfg)
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		_ = conn.Close(ctx)
+	})
+	return conn
+}
+
+// masterUserPasswordSecret is used to unmarshal an AWS Secrets Manager
+// master user and password secret.
+type masterUserPasswordSecret struct {
+	User string `json:"username"`
+	Pass string `json:"password"`
+}
+
+// getMasterUserPassword is a helper that fetches a db master user and password
+// from AWS Secrets Manager.
+func getMasterUserPassword(t *testing.T, ctx context.Context, secretID string) string {
+	t.Helper()
+	secretVal := getSecretValue(t, ctx, secretID)
+	var secret masterUserPasswordSecret
+	if err := json.Unmarshal([]byte(*secretVal.SecretString), &secret); err != nil {
+		// being paranoid. I don't want to leak the secret string in test error
+		// logs.
+		require.FailNow(t, "error unmarshaling secret string")
```

Comment:
I prefer the formatting that testify provides, especially in helper functions, since it provides a trace. For example:
```go
func TestFatality(t *testing.T) {
	t.Run("one", func(t *testing.T) {
		helperFatal(t)
	})
	t.Run("two", func(t *testing.T) {
		testifyHelperFail(t)
	})
}

func helperFatal(t *testing.T) {
	t.Helper()
	t.Fatal("big fail!")
}

func testifyHelperFail(t *testing.T) {
	t.Helper()
	require.FailNow(t, "big fail!")
}
```

```fish
$ go test . -run TestFatality
--- FAIL: TestFatality (0.00s)
    --- FAIL: TestFatality/one (0.00s)
        main_test.go:545: big fail!
    --- FAIL: TestFatality/two (0.00s)
        main_test.go:548: 
            	Error Trace:	/Users/gavin/proj/gosandbox/main_test.go:559
            	           				/Users/gavin/proj/gosandbox/main_test.go:548
            	Error:      	big fail!
            	Test:       	TestFatality/two
```

Reply:
That's fair.